app/src/main/java/org/oppia/android/app/devoptions/DeveloperOptionsViewModel.kt

```diff
@@ -29,7 +30,11 @@ class DeveloperOptionsViewModel @Inject constructor(
     val itemViewModelList: MutableList<DeveloperOptionsItemViewModel> =
       mutableListOf(DeveloperOptionsModifyLessonProgressViewModel())
     itemViewModelList.add(DeveloperOptionsViewLogsViewModel())
-    itemViewModelList.add(DeveloperOptionsOverrideAppBehaviorsViewModel())
+    itemViewModelList.add(
+      DeveloperOptionsOverrideAppBehaviorsViewModel(
+        forceCrashListener = forceCrashListener
```

Comment:
Nit: the named argument isn't needed here since the variable name is fully self-documenting.

Reply:
Removed named arguments.